backend/.claude/skills/run-tests/SKILL.md

```
@@ -0,0 +1,51 @@
+
+| 대상 입력                    | 실행 명령                                        |
+|--------------------------|----------------------------------------------|
+| `:module` (콜론으로 시작)      | `./gradlew :module:test`                     |
+| `package.pattern` (점 표기) | `./gradlew test --tests "package.pattern.*"` |
+| 비어 있음                    | `./gradlew test`                             |
+
+## Step 2: 에이전트 실행
+
+에이전트 프롬프트는 다음과 같다:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)